src/types/index.ts

```diff
@@ -367,8 +367,8 @@ export interface ITx {
   recipientId?: string
   senderId?: string
   selectedTokenContractId?: string
-  tag?: Tag;
-  type: keyof typeof Tag,
+  tag?: Tag
+  type: string
```

Comment:
I think this change is bad - we need to have explicit list of possible values.

Reply:
Fixed it. Added comment too.